stock-price-checker/routes/api.js

```diff
@@ -19,7 +19,8 @@ module.exports = function (app) {
     .get(function (req, res){
       var stock = req.query.stock;
       var like = req.query.like || false;
-      var reqIP = req.connection.remoteAddress;
+      var requestIP = req.get('x-forwarded-for') || req.ip;
+      var reqIP = requestIP.split(",")[0];
```

Comment:
Tested some regex from [this SO post](https://stackoverflow.com/questions/48767382/anonymize-ipv4-and-ipv6-addresses-with-php-preg-replace) and it seems to work for both IPv4 and IPv6 addresses:

```suggestion
      var requestIP = req.get('x-forwarded-for') || req.ip;
      var reqIP = requestIP.split(",")[0];
      reqIP = reqIP.replace(/\.\d*$/, '.0'); // ipv4
      reqIP = reqIP.replace(/[\da-f]*:[\da-f]*$/, '0:0'); // ipv6
```

There's probably a smarter way to do this, but the code above works in a pinch.

Reply:
I think we would need to ignore the case on those v6 versions